UnitySetup/UnitySetup.psm1

```diff
@@ -1761,7 +1761,15 @@ function Start-UnityEditor {
         [parameter(Mandatory = $false)]
         [switch]$Wait,
         [parameter(Mandatory = $false)]
-        [switch]$PassThru
+        [switch]$PassThru,
+        [parameter(Mandatory = $false)]
+        [string]$CacheServerEndpoint,
+        [parameter(Mandatory = $false)]
+        [string]$CacheServerNamespacePrefix,
+        [parameter(Mandatory = $false)]
+        [switch]$CacheServerDisableDownload,
+        [parameter(Mandatory = $false)]
+        [switch]$CacheServerDisableUpload
```

Comment:
Can you add a description of these parameters in the function header comment? perhaps even example?

Reply:
An example in the readme would be great!

Reply:
Thanks for the suggestions, I agree it's much clearer now with the examples.